docs/learning/backend/Workspace.md

```diff
@@ -54,15 +102,13 @@ For example:
 "vibration-map/filters/prefabricated"
 ```
 
-`SettingName`s must be valid [JavaScript property names](https://developer.mozilla.org/en-US/docs/Glossary/property/JavaScript), but should not contain periods or spaces.
-
-### SettingSchema
+Are SettingNames from the `energyAnalysis`, `iot-scan-visualization`, and `vibration-map` SettingSchemas.
 
-A single `Setting` is defined according to the rules of [JSON Schema](https://json-schema.org/) via a [SettingSchema]($backend). The primary objective of creating a [SettingSchema]($backend) is to advertise the existence, meaning, and "form" of a Setting. Users supply values for setting using a settings editor, and are presented with the information from `SettingsSchema`s to guide their choices. Also, `SettingSchema`s may also supply a default value so users can understand what happens if they don't provide a value for a Setting.
+`SettingName`s must be valid [JavaScript property names](https://developer.mozilla.org/en-US/docs/Glossary/property/JavaScript), but may not contain periods or spaces.
```

Comment:
When you say setting names must be "valid JS property names" do you really mean JS identifiers (i.e., legal variable names)? Any valid JS string is a valid JS property name. JS identifiers can't start with a number, contain whitespace, contain characters like "-", "+", etc. If that is your intent, what's the motivation for those restrictions?

Reply:
I think setting names should be "simple".

Reply:
I don't see any code enforcing these restrictions, or a clear definition of what the restrictions actually are.